content/docs/tasks/policy-enforcement/denial-and-list/index.md

```diff
@@ -168,6 +168,17 @@ subnet.
     not whitelisted`
 
 ## Cleanup
+* Remove the Mixer configuration for simple denials:
+
+    {{< text bash >}}
+    $ kubectl delete -f @samples/bookinfo/policy/mixer-rule-deny-label.yaml@
+    {{< /text >}}
+    
+    If you are using Istio 1.1.2 or prior:
```

Comment:
I think this is against docs review policy (@geeknoid correct me if I'm wrong please).  I think we want to take the position that when operators read the documentation they are accessing documentation for the specific release they are reviewing.  As this is a master PR (which will seed release-1.2 branch when @geeknoid branches the repo), 1.1.2 is not relevant.  If you want 1.1.2 commentary, please submit a PR to release-1.1 and operators will be able to access the 1.1.2 documentation directly.

Reply:
Removed 1.1.2 or prior comment..